payloads/external/edk2/Makefile

```diff
@@ -121,10 +121,14 @@ endif
 ifeq ($(CONFIG_EDK2_TPM_ENABLE),y)
 BUILD_STR += -D TPM_ENABLE=TRUE
 endif
+# NETWORK_LAN_ROM		= FALSE
```

Comment:
Commit says only about disabling MTRR. Do we also need that line here?

Reply:
Forgot to add the comment with the default value in previous commits/PRs for this option. Good to have but not needed.

Reply:
Maybe it should be a separate commit then, if note related to the MTTR option? Would make it easier when rebasing and cleaning up patches.

Reply:
Made it a separate commit

Reply:
Thanks